docs/topics/media-pipeline.rst

```diff
@@ -197,6 +197,25 @@ For self-hosting you also might feel the need not to use SSL and not to verify S
     AWS_USE_SSL = False # or True (None by default)
     AWS_VERIFY = False # or True (None by default)
 
+
+Custom Amazon S3 headers in FileStorage
+
+Steps to pass **SSEKMSKeyId** and **ServerSideEncryption** to **scrapy** FilesPipeline using **AWS S3**
```

Comment:
This reads very specific to those 2 headers.

Reply:
Done, resolved them and made them generalised